### PR TITLE
Add banner

### PR DESCRIPTION
**Pull request by thiamsantos**
Ref #116. AIt will prepend a banner on the final bundle with the following format:

```js
/*!
 * banner v1.0.0
 * Zero-configuration bundler for tiny JS libs, powered by Rollup.
 *
 * https://github.com/developit/microbundle
 *
 * @author Jason Miller <jason@developit.ca> (http://jasonformat.com)
 * @license MIT
 */
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "rollup-plugin-typescript2": "^0.13.0",
     "rollup-plugin-uglify": "^3.0.0",
     "sade": "^1.4.0",
+    "stringify-author": "^0.1.3",
```
Comment (ForsakenHarmony) [inline package.json]: Sorry, but no dependency hell

File: src/index.js [hunk 2]
```diff
@@ -287,7 +288,7 @@ function createConfig(options, entry, format, writeMeta) {
 				options.compress !== false && [
 					uglify({
 						sourceMap: true,
-						output: { comments: false },
+						output: { comments: /^!/ },
```
Comment (ForsakenHarmony) [inline src/index.js]: What's this gonna do to other comments, should probably depend on the option

File: src/index.js [hunk 3]
```diff
@@ -342,9 +343,25 @@ function createConfig(options, entry, format, writeMeta) {
 			},
 			format,
 			name: options.name,
-			file: resolve(options.cwd, (format === 'es' && moduleMain) || (format === 'umd' && umdMain) || cjsMain)
+			file: resolve(options.cwd, (format === 'es' && moduleMain) || (format === 'umd' && umdMain) || cjsMain),
+			banner: createBanner(options)
```
Comment (ForsakenHarmony) [inline src/index.js]: Can we put the conditional here?

( options.banner &&...)

**Comment by thiamsantos** [comment]
@ForsakenHarmony Hi! Do you that this feature is a good addition to microbundle? It was so long ago, that I don't even remember why I thought it was a good idea. But if it is a really great feature to add I would be happy to make the changes suggested. ;)

**Comment by ForsakenHarmony** [comment]
I don't personally have a use for it tbh

it's up to you

**Comment by thiamsantos** [comment]
@ForsakenHarmony I will close this PR for now. If anyone in the future request this feature or have a use for it I'll be happy to take another shot on this.

**Pull request closed without merging**